app/views/devise/registrations/new.html.erb

```diff
@@ -1,28 +1,56 @@
-<h2>Sign up</h2>
+<div class="register_container">
+<h2 class="hello_h2">Hello!</h2>
+<p>Create New account here. </p>
+
 
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
+
+  <div class="field">
+    <%= f.text_field :name, autofocus: true, autocomplete: "email",class: 'form_field',placeholder:"Full Name" %>
+  </div>
+  
+   <div class="field">
+    <%= f.text_field :phone, autofocus: true, autocomplete: "email",class: 'form_field',placeholder:"Phone" %>
```

Comment:
There's no point on having multiple fields with `autofocus: true`. There should be only one. That way the browser can put the focus there once the page loads.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#attr-autofocus

Reply:
It's true